src/index.html

```diff
@@ -1,16 +1,49 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search-bar">
+      <form
+      action="#"
+      method="post"
+      class="search-bar-form"
+      data-qa="big"
+      >
```

Comment:
attributes indentation here and for the rest of the code:
```suggestion
      <form
        action="#"
        method="post"
        class="search-bar-form"
        data-qa="big"
      >
```